internal/characters/kazuha/skill.go

```diff
@@ -36,7 +36,14 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 	hold := p["hold"]
 
 	c.a1Absorb = attributes.NoElement
-	c.a1AbsorbCheckLocation = combat.NewCircleHit(c.Core.Combat.Player(), 1.5)
+
+	radius := 5.0
+
+	if hold >= 1 {
+		radius = 9
+	}
+
+	c.a1AbsorbCheckLocation = combat.NewCircleHit(c.Core.Combat.Player(), radius)
```

Comment:
pls remove the c.a1AbsorbCheckLocation and qAbsorbCheckLocation definition from kazuha.go